### PR TITLE
Update tests for Add command

### PR DESCRIPTION
**Pull request by Bearypop**
Update test files for Add command and resolve checkstyle issues.

### CODE REVIEW

**Review by maj0-0 — COMMENTED** [review]
AddCommand Tests seem to pass now, LGTM!

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java [hunk 1]
```diff
@@ -157,6 +159,11 @@ public ObservableList<Person> getFilteredPersonList() {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
```
Comment (neyapraveen) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:159]: Tests seem to pass, LGTM!

**Review by neyapraveen — COMMENTED** [review]
AddCommand Tests LGTM!

**Review by wnchan — COMMENTED** [review]
LGTM!

**Pull request merged**